### ISSUE TITLE
Execution error

### ISSUE THREAD

**Issue by hoirulbasori** [iryoku/separable-sss#3]
I have successfully compile the demo sample of separable SSS, however when i run the program i got error"failed creating the Direct3D device"? how this happen?

i have installed directx11 run in alienware with NVIDIA geforce 660M

could you please assist me in this issue?

Thanks
Hoirul

**Comment by iryoku** [comment]
Hi,

This is probably because you are on Windows 8. You need the Windows SDK 
for the demo to run on Windows 8.

Cheers!
Jorge

**Comment by hoirulbasori** [comment]
Dear Jorge,
Many Thanks for your reply.
i have installed the Windows SDK, so my compilation is no error.
in my debug folder, the demo executable file is successfully created. but when i try to run it by double click. i found this errorv"failed creating the Direct3D device".

Thank you very much. hope to hear reply from you.

Thanks
Hoirul

**Comment by hoirulbasori** [comment]
Sorry forgot to highlight, i run th eproject on windows 7 64 bit SP1

**Comment by iryoku** [comment]
Which GPU has your computer?

**Comment by hoirulbasori** [comment]
Geforce GTX 660M
2GB Ram

**Comment by iryoku** [comment]
Can you run various DirectX SDK demos and see if they work properly?

**Comment by hoirulbasori** [comment]
yes i have tried Directx sample executable for directx11 or 10, everything it works perfectly. I run the executable version of separable SSS, and it works, but when i run it from source code, its have an error "failed creating the Direct3D device"

**Comment by 0x779** [comment]
Have you thought about debugging it?

**Comment by iryoku** [comment]
For other similar project, I've seen the same problem recently in a dual GPU laptop (Intel/Nvidia), and only happened in debug builds. My other two dev machines seems to work fine; I believe it is problem in DXUT.
